lib/Basics/errors.dat

```diff
@@ -167,6 +167,7 @@ ERROR_REPLICATION_REPLICATED_LOG_INVALID_TERM,1425,"an invalid term was given","
 ERROR_REPLICATION_REPLICATED_LOG_UNCONFIGURED,1426,"log participant unconfigured","Will be raised when a participant is currently unconfigured, i.e. neither a leader nor a follower."
 ERROR_REPLICATION_REPLICATED_STATE_NOT_FOUND,1427,"replicated state {id:} of type {type:} not found","Will be raised when a specific replicated state was not found."
 ERROR_REPLICATION_REPLICATED_STATE_NOT_AVAILABLE,1428,"replicated state {id:} of type {type:} is unavailable","Will be raised when a specific replicated state was accessed but is not (yet) available."
+ERROR_REPLICATION_REPLICATED_LOG_SUBSEQUENT_FAULT,1429,"operation aborted because a previous operation failed","Will be raised when an operation is aborted to a cascading failure."
```

Comment:
```suggestion
ERROR_REPLICATION_REPLICATED_LOG_SUBSEQUENT_FAULT,1429,"operation aborted because a previous operation failed","Will be raised when an operation is aborted due to a cascading failure."
```